tests/integration/expected_test_results/api_v3_product_read/get-existing-product-gs1-caret.json

```diff
@@ -104,7 +104,7 @@
             "origins_of_ingredients" : {
                "aggregated_origins" : [
                   {
-                     "epi_score" : 0,
```

Comment:
is this expected?

Reply:
It's an artefact of the tests, Perl has this strange habit of changing the type of scalars according to the last operation that was done on them, and for some reason from one test run to another we get differences.